docs/application/dotnet/guides/connectivity/webview.md

```diff
@@ -0,0 +1,166 @@
+
+## Prerequisites
+
+Ensure the following to enable WebView in your application:
+
+1. To use the `Tizen.WebView` namespace, create the `ElmSharp-Beta` project using the [Project Wizard](https://developer.tizen.org/development/visual-studio-tools-tizen/tools/project-wizard) of Visual Studio.
+
+2. To use the class of the `Tizen.WebView` namespace, you must request for permission by adding the following privileges to the `tizen-manifest.xml` file, as shown in the following code snippet:
+
+   ```
+   <privileges>
+     <!--To use the Internet connection-->
+     <privilege>http://tizen.org/privilege/internet</privilege>
+
+     <!--To access media storage-->
+     <privilege>http://tizen.org/privilege/mediastorage</privilege>
+
+     <!--To access, read, and write to the external storage-->\
+     <privilege>http://tizen.org/privilege/externalstorage</privilege>
+   </privileges>
+   ```
+
+3. To use the methods and properties of the `Tizen.WebView` namespace, include it in your application, as shown in the following code snippet:
+    ```
+    using Tizen.WebView;
+    ```
+
+<a name="initialize"></a>
+## Initializing and Finalizing WebView
+
+To Initialize and Finalize WebView, follow these steps:
+
+1. To use the `Tizen.WebView` namespace in your application, call the `Tizen.WebView.Chromium.Initialize` method before the start of the main loop, as shown in the following code snippet:
+
+    ```
+    Elementary.Initialize();
+    Elementary.ThemeOverlay();
+    Chromium.Initialize();
+    App app = new App();
+    app.Run(args);
+    ```
+
+2. To clean up the resources allocated by your application, call the `Tizen.WebView.Chromium.Shutdown` method after the end of the main loop, as shown in the following code snippet:
+
+    ```
+    App app = new App();
+    app.Run(args);
+    Chromium.Shutdown();
+    ```
+
+<a name="create"></a>
+## Creating WebView Object
+
+To create a WebView object, use the constructor of `Tizen.WebView.WebView` with `ElmSharp.Window`, as shown in the following code snippet:
+
+```
+var webView = new WebView(window)
+{
+    AlignmentX = -1,
+    AlignmentY = -1,
+    WeightX = 1,
+    WeightY = 1
+};
+webView.Show();
+```
+
+<a name="load"></a>
+## Loading Web Page
+
+To load a web page, use the `Tizen.WebView.WebView.LoadUrl` method with an appropriate URL, as shown in the following code snippet:
+
+```
+webView.LoadUrl("https://www.tizen.org/");
+```
+
+<a name='event'></a>
+## Handling Load Events
+
+The following table lists the loading events provided by the `Tizen.WebView.WebView` class:
+
+**Table: Loading Events**
+
+| Event            | Description                                                       | Argument                                 |
+|------------------|-------------------------------------------------------------------|------------------------------------------|
+| `LoadStarted`    | Event occurs when the application starts to load.                 | -                                        |
```

Comment:
@younghajung 
Kindly replace with the following text from line 100 -103 with:

| Event            | Description                                                       | Argument                                 |
|------------------|-----------------------------------------|:------------------------------------------:|
| `LoadStarted`    |  This event occurs when loading starts. |   -  |            
| `LoadFinished`   |  This event occurs when loading finishes. |  - | 
| `LoadError`      |  This event occurs when an error appears while loading. |       Tizen.WebView.SmartCallbackLoadErrorArgs |

Reply:
Done